data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report6_fitdata.py

```diff
@@ -549,6 +552,53 @@
         ignore_index=True)
 df1.T
 
+# # Verify that all records with invalid level values are being dropped from sleep_level table
+#
+# DC-2606
+#
+# Queries the sandbox table for the corresponding cleaning rule (created in DC-2605) and outputs any records that are being dropped due to invalid level values. It also outputs any records in the sleep_level table that should have been dropped but were not.
```

Comment:
This line has 258 characters.  How about breaking it somewhere between 100 and 110 characters in length.